CODING_STYLE.md

```diff
@@ -1,8 +1,8 @@
 LXC Coding Style Guide
 ======================
 
-In general the LXC project follows the Linux kernel coding style.  There are
-however are a few differences, these are outlined in this document.
+In general the LXC project follows the Linux kernel coding style.  Hwoever,
```

Comment:
@brauner typo

Reply:
No, that's how this word is written. For sure. :D

Reply:
fixed